getting-started-tutorial/getting-started-tutorial.ipynb

```diff
@@ -1556,8 +1549,49 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "project.set_function('hub://describe', 'describe')\n",
-    "describe = project.func('describe').apply(mount_v3io())"
+    "describe = project.set_function('hub://describe', 'describe').apply(mount_v3io())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### View the function documentation"
```

Comment:
Our convention is to use title case (not sentence case) in notebook headings (it's currently not consistent across all NBs, but it's applied in this NB).

Reply:
Fixed in latest commit